source/plugins/ruby/constants.rb

```diff
@@ -125,7 +125,7 @@ class Constants
   CI_EXTENSION_VERSION = "1"
   #Current CI extension config size is ~5KB and going with 20KB to handle any future scenarios
   CI_EXTENSION_CONFIG_MAX_BYTES = 20480
```

Comment:
since the extension config will have the syslog in it, dont we need to bump value of this CI_EXTENSION_CONFIG_MAX_BYTES ? did you check what's the extension size with syslog?

Reply:
why? we have not changed CI config, the syslog additions are not part of the CI extension config in the DCR config and this variable denotes the max size of just the CI part of the config